src/core/include/openvino/core/attribute_adapter.hpp

```diff
@@ -34,6 +35,12 @@ class OPENVINO_API ValueAccessor<void> {
     /// as_type.
     virtual const DiscreteTypeInfo& get_type_info() const = 0;
     virtual ~ValueAccessor() = default;
+    virtual ov::Any get_as_any() {
```

Comment:
Why do we need this?
Maybe should we throw ov::Exception?

Reply:
this method is needed to handle custom user attributes as ov::Any
user can map attribute names and values and do not specify a concrete type, e.g.
`{ "attr_name_1" : 5, "attr_name_2" : "string", "attr_name_3" : enum}`
actually it's map<std::string, **ov::Any**>

I will add more tests to cover most common types

Reply:
the tests have been added
changed to ov::Exception